src/components/tx/TxData.js

```diff
@@ -518,14 +523,15 @@ class TxData extends React.Component {
     };
 
     const renderTwins = () => {
+      const conflictNotTwin = this.props.meta.conflict_with.length;
       if (!this.props.meta.twins.length) {
         return null;
       }
       return (
         <div>
           This transaction has twin{' '}
           {helpers.plural(this.props.meta.twins.length, 'transaction', 'transactions')}:{' '}
-          {renderListWithLinks(this.props.meta.twins, true)}
+          {renderListWithLinks(conflictNotTwin, true)}
```

Comment:
question: The parameter was changed from an array to an integer. Why was that needed?

Reply:
Ended up being modified along with the other renderListWithLinks by mistake. Fixed. Please review.